bindata/network/ovn-kubernetes/monitor.yaml

```diff
@@ -1,4 +1,4 @@
-
+---
```

Comment:
nit: the separator seems un-necessary since there is nothing before.

Reply:
@abhat yeah that part was in Juan's original PR